flybywiresim/aircraft
# chore: upgrade msfssdk to v0.7

Fixes #[issue_no]

## Summary of Changes

Full changelog https://github.com/microsoft/msfs-avionics-mirror/releases/tag/framework-v0.7.0
The main upside for us is FSComponent support for the SVG tspan element. We had been using some hacks before to work around the lack of support, and these are undone in this PR.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- check that the minimums field on the FMA works correctly,
- check that the vertical mode slot on the FMA works correctly,
- check that EWD warnings and memos work correctly,
- check that instruments generally function correctly.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
